instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -342,24 +351,38 @@ def _populate_span(
         if self._db_api_integration.capture_parameters and len(args) > 1:
             span.set_attribute("db.statement.parameters", str(args[1]))
 
+    def get_operation_name(self, cursor, args):  # pylint: disable=no-self-use
+        if args and isinstance(args[0], str):
+            return args[0].split(" ")[0]
```

Comment:
```suggestion
            return args[0].split()[0]
```

I think this (no `sep` arg) might have better behavior for variety of SQL formattings e.g.
```py
# leading whitespace
cursor.execute("""
SELECT * FROM foo
""")

# newline sep
cursor.execute(
  """SELECT
         *
       FROM foo
  """
)
```